doc/examples/example_cmake.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+export SRC_DIR=`pwd`
+export BLD_DIR=/tmp/bld
+
+mkdir $BLD_DIR
+
+#BEGIN cmake_build
+cd $BLD_DIR
+cmake $SRC_DIR
+make VERBOSE=1
+#END cmake_build
+
+#BEGIN cmake_run
+./program
+#END cmake_run
+
+#BEGIN cmake_gcovr
+gcovr -r $SRC_DIR $BLD_DIR
```

Comment:
Does the 'should' mean "the working directory will be `$BLD_DIR` or does it mean "`gcovr` has to be invoked from `$BLD_DIR`?" I'd prefer to leave `$BLD_DIR` in place, to remind users of its semantic meaning, rather than what it happens to evaluate to.

Reply:
“Gcovr should be invoked from the build dir” means that gcovr may or may not work when invoked from outside of the build dir. This has to do with the heuristics for finding the correct directory to run gcov from.

I have updated the example to explicitly `cd` into the build dir first.